### ISSUE TITLE
Modifying the library locally

### ISSUE THREAD

**Issue by katicstefan** [ideoforms/AbletonOSC#126]
Hi, I'm working on a personal project that I plan to release to the public when it's done and it's using this library and potentially pylive too. Basically, I have to access some LOM endpoints which don't work with current AbletonOSC version (Application bugfix_version property, Song name and Song file_path).
I experimented by modifying the application.py and song.py by adding functions and properties from LOM to your code, and testing it through the console util it retrieved the data I need.
I don't have a problem patching in my custom code in the future when you update the library, but I'm worried if I'm allowed to do it since the app will have to use your library and I'm modifying it.
Sorry if this is a dumb question, I basically started this project today and I'm happy I even got this thing to work with how little official docs or resources are available, on top of that I'm kinda new to python so I'm struggling, plus I never in my previous webdev projects had to modify a library to get it to do what I need.

**Comment by ideoforms** [comment]
Yes, feel free to submit a Pull Request with any new functionality. I've added some guidelines on doing so in a PR template.
